storage/innobase/handler/ha_innodb.cc

```diff
@@ -15252,6 +15252,16 @@ ha_innobase::check(
 		m_prebuilt->need_to_access_clustered =
 			!!(check_opt->flags & T_EXTEND);
 
+		if (m_prebuilt->need_to_access_clustered && srv_use_scn) {
+			/* trx id on secondary index page can't be used to check visibility while scn is used, so disable it and print warning */
```

Comment:
How would MVCC on secondary indexes work if `innodb_use_scn=ON`? The commit message does not mention anything about it.

Reply:
The MVCC  on secondary index remains the same as before. Firstly call `ReadViewBase::sees` with the `max_trx_id` obtained from the page. If the result is false, the next step is to examine the clustered index.